tool/tbot/main.go

```diff
@@ -365,3 +372,23 @@ func handleSignals(log logrus.FieldLogger, cancel context.CancelFunc, reloadCh c
 		}
 	}
 }
+
+func setupLogger(debug bool, format string) error {
+	level := logrus.InfoLevel
+	if debug {
+		level = logrus.DebugLevel
+	}
+	utils.InitLogger(utils.LoggingForDaemon, level)
+
+	switch format {
+	case config.LogFormatJSON:
+		formatter := &utils.JSONFormatter{}
+		logrus.SetFormatter(formatter)
+	case config.LogFormatText, "":
+	// Nothing to do, this is the default set up by utils.InitLogger
+	default:
+		return trace.BadParameter("unsupported log format %q", format)
```

Comment:
Nit: this should be impossible in practice, so maybe we can just stay with `text` if it ever happens?

Reply:
Mm, I'd rather throw an error here in case we (as developers) screw something up and we don't notice because it starts silently falling back to text.